### DISCUSSION TITLE
Recent Mandates by Google Play to make apps target Android 13

### DISCUSSION THREAD

**Discussion by destynn** [dotnet/maui#17747] [Q&A] [answered]
Hey team,

I am wondering if I am possibly misunderstanding the latest mandates that all new apps target Android 13 (API 33.0)

When I submitted my app to the store it made me unable to target API's lower than 33 or else it would throw an error.

![Untitled](https://github.com/dotnet/maui/assets/48139153/453bd899-e767-4778-85b4-6685855aa5ef)

Am I actually able to change the minimum target build as long as the main target is android 13 (API 33.0) or are we now forced to only support devices that can go up to android 13 (API 33.0). If I changed the supported OS to 21.0 (which is what I had it in a previous version) it only now recently gives me an error message when submitting signed build to google play.

I feel like this is a very simple/dumb question but for some reason their jargon for it escapes my comprehension for what to do about those with older devices. Is it standard to just leave them in the dust? (It feels weird to have someone who WAS able to use this on their device that now suddenly can't even download it just because they couldn't update their OS on an older device).

Thanks for either pointing me directly to the documentation/message that will clear this part of it up, or for answering me directly either one is fine.

NOTE: I've already read: https://support.google.com/googleplay/android-developer/answer/11926878?hl=en and subsequent pages but it doesn't really talk about people/devices who can't upgrade to that version.

Thanks,

**Comment by developer9969** [comment]
Hi 
I had the same problem with Xamarin - Min target does not matter!! what matter is the target framework.. Do you have any dependencies elsewhere?

Are you using androidX dependencies that do  not support 33?

**Reply by destynn** [reply]
Thanks for replying!

No, i dont have any dependencies that I can tell.

I swear I had an issue where where I changed the bottom thing in red to 21.0.

Then Google play gave me an error when I uploaded it. See below screenshot of it. (Also try to rule out any mistake I made)

![image](https://github.com/dotnet/maui/assets/48139153/8a17325c-0dbc-418c-817c-768574e4e2f1)

Maybe there's a different setting in the project file that allows me to establish what the minimum target is.

**Reply by developer9969** [reply]
just stating the obvious  
check your visual studio proj file and most importanly android manifest
`    <uses-sdk android:minSdkVersion="29" android:targetSdkVersion="33" />`  

![image](https://github.com/dotnet/maui/assets/16970109/dd084559-136d-482f-8e1c-a9be6392873d)

**Comment by destynn** [comment, marked as answer]
Alright I'm trying again, wish me luck!

![image](https://github.com/dotnet/maui/assets/48139153/3c36cdb4-3b8f-4f24-9526-b5cc8391da05)

I'm not doing anything differently than I did last time? (Is my issue maybe because I'm trying to target REALLY OLD android APIs too?)

![image](https://github.com/dotnet/maui/assets/48139153/6c8d517b-d959-43ea-833d-d6ab1d2e8021)
